source/expressions.tex

```diff
@@ -4166,10 +4166,10 @@
 The unary \tcode{*} operator performs \defn{indirection}:
 \indextext{dereferencing|see{indirection}}%
 the expression to which it is applied shall be a pointer to an object
-type, or a pointer to a function type and the result is an lvalue
+type, or a pointer to a function type and the resulting expression is an lvalue
```

Comment:
"resulting expression" sounds like a static property, but we're actually talking about the runtime result value (which is an lvalue) here.